deploy/3-azure-iot-operations.sh

```diff
@@ -88,4 +88,4 @@ az deployment group create \
     --parameters customLocationName=$custom_location_name \
     --no-prompt
 
-echo "Finished deploying Azure IoT Operations Preview components to cluster $CLUSTER_NAME in resource group $RESOURCE_GROUP"
\ No newline at end of file
+echo "Finished deploying Azure IoT Operations ${cli_pinned_version} components to cluster $CLUSTER_NAME in resource group $RESOURCE_GROUP"
```

Comment:
is this really the version or only the CLI version? just to be aligned with my version comment above

Reply:
Thank you Marcel :)